Tests/Services/NfieldSurveysServiceTests.cs

```diff
@@ -223,28 +225,169 @@ public void TestUploadInterviewerInstructionsAsync_ValidByteArray_FileUpload()
         [Fact]
         public void TestQuotaQueryAsync_ServerReturnsQuery_ReturnsListWithQuotaLevel()
         {
-            const string levelId = "LevelId";
-            const string name = "Name";
-
             var expectedQuotaLevel = new QuotaLevel
             {
-                Id = levelId,
-                Name = name
+                Id = LevelId,
+                Name = Name
             };
             var mockedNfieldConnection = new Mock<INfieldConnectionClient>();
             var mockedHttpClient = CreateHttpClientMock(mockedNfieldConnection);
             mockedHttpClient
-                .Setup(client => client.GetAsync(new Uri(ServiceAddress, "Surveys/1/Quota")))
+                .Setup(client => client.GetAsync(new Uri(ServiceAddress, $"Surveys/{SurveyId}/Quota")))
                 .Returns(CreateTask(HttpStatusCode.OK, new StringContent(JsonConvert.SerializeObject(expectedQuotaLevel))));
 
             var target = new NfieldSurveysService();
             target.InitializeNfieldConnection(mockedNfieldConnection.Object);
 
-            var actualQuotaLevel = target.QuotaQueryAsync("1").Result;
+            var actualQuotaLevel = target.QuotaQueryAsync(SurveyId).Result;
             mockedHttpClient.Verify(hc => hc.GetAsync(It.IsAny<Uri>()), Times.Once());
             Assert.Equal(expectedQuotaLevel.Id, actualQuotaLevel.Id);
             Assert.Equal(expectedQuotaLevel.Name, actualQuotaLevel.Name);
+        }
+
+        [Fact]
+        public void TestQuotaTargetsQueryAsync_ServerReturnsQuery_ReturnsListWithQuotaFrame()
+        {
+            const string QuotaFrameId = "quotaId";
```

Comment:
Most (all?) of these constants are only used once in the test (same for the next one). From my point of view, adding them as independent variables in this case doesn't help and they could be removed